scripts/ucsc/create-ucsc-packages.py

```diff
@@ -198,6 +197,12 @@ def program_subdir(program, names):
     'gfServer',
 ]
 
+# A list of programs which have problems to build on linux-aarch64
+SKIP_AARCH64 = [
```

Comment:
I think at some point we should have a global list, that script like this one can consume.